guideforms.py

```diff
@@ -436,7 +439,7 @@
         help_text='Android WebView:<br/>' + SHIPPED_WEBVIEW_HELP_TXT),
 
     'prefixed': forms.BooleanField(
-        required=False, initial=False, label='Prefixed?'),
+        required=False, initial=False, label='WebKit prefixed?'),
 
```

Comment:
Yes, we should make this change, but it's getting increasingly hard to justify the existence of this question nowadays. It's been quite a while since we all agreed prefixing things was a bad idea, I can't remember the last time prefixing something was even suggested...

Reply:
Filed an issue on this: https://github.com/GoogleChrome/chromium-dashboard/issues/1036